### DISCUSSION TITLE
Showing wrong match

### DISCUSSION THREAD

**Discussion by harjasae2001** [cssbattle/forum#379] [Q&A] [answered]
<h1>Despite having 0 difference from the original image, it is still showing only 96% match <h1>

![Screenshot from 2024-01-04 09-13-45](https://github.com/cssbattle/forum/assets/83627055/af3bd9ea-1e3a-4e99-b1a0-3e7294c2add1)

**Comment by SelenIT** [comment, marked as answer]
The backend currently uses the Chromium 119 browser engine to check solutions. If you use a non-Blink-based browser (e.g. Firefox or Safari), it's better to switch to the Blink-based one (like Chrome, Edge, or Opera).

If you are using the latest Chrome (and the like), check your selector syntax. Chrome 120 supports the newest CSS Nesting spec that allows interchanging CSS properties with nested type selectors (like `* {  color: red; p { color: green }}`), but Chromium 119 supports only the older version of the Nesting spec that required nested selectors to start with a combinator or the `&` character (i.e. `* {  color: red; > p { color: green }}` or `* {  color: red; & p { color: green }}`).
